### PR TITLE
Create DRPolicy page indentation and tooltip

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: Gowtham Shanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
image

File: packages/mco/components/data-policies/create-dr-policy/create-dr-policy.tsx [hunk 1]
```diff
@@ -422,12 +424,29 @@ export const CreateDRPolicy: React.FC<ReRouteResourceProps> = ({
...
           <SelectClusterList
             selectedClusters={selectedClusters}
             setSelectedClusters={setSelectedClusters}
           />
         </FormGroup>
+        <FormGroup fieldId="policy-name">
+          <Alert
+            className="co-alert mco-create-data-policy__alert"
+            title={t(
+              'Openshift Data Foundation 4.11 or above must be installed on the managed clusters to setup connection for enabling replication/mirroring'
```
Comment (afreen23) [inline packages/mco/components/data-policies/create-dr-policy/create-dr-policy.tsx:444]:
```suggestion
              'Data Foundation 4.11 or above must be installed on the managed clusters to setup connection for enabling replication/mirroring'
```

**Review by afreen23 — CHANGES_REQUESTED** [review]

File: packages/mco/components/data-policies/create-dr-policy/create-dr-policy.tsx (hunk 1 above)
Comment (GowthamShanmugam) [reply packages/mco/components/data-policies/create-dr-policy/create-dr-policy.tsx:444]: Ack

**Review by afreen23 — APPROVED** [review]

**Pull request merged**